src/chrome/content/rules/Despora.de.xml

```diff
@@ -1,10 +0,0 @@
-<ruleset name="Despora.de">
-
-	<target host="despora.de" />
-	<target host="www.despora.de" />
```

Comment:
This target is not yet preloaded (only `^` is).

Reply:
This is because only the main domain is listed, right? Like the others.

Reply:
The host is not preloaded with the `includeSubDomains` flag in Firefox. Also, the server uses an invalid HSTS configuration: https://hstspreload.org/?domain=despora.de
So this host should be removed at all (#13682).

Reply:
Okay, it's not surprising the script got confused here. I opened an issue to fix this issue directly with the website: https://github.com/despora/diaspora/issues/12.

Reply:
@Bisaloo could you revert the removal of `Despora.de.xml` for now that we can ship this in #17157?

Reply:
@J0WI, done